hr_security_adj/views/hr_views.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr.menu_hr_employee_payroll" model="ir.ui.menu">
+        <field name="groups_id" eval="[(6, 0, [ref('group_hr_operator')])]" />
+    </record>
+</odoo>
```

Comment:
Why is this part necessary?

Reply:
employee view is only allowed to show for hr_group_user. So, we need to add operator group.https://github.com/odoo/odoo/blob/8221bef0a023d33476bdaca868f2ff0991f12a0d/addons/hr/views/hr_views.xml#L18-L30